.eslintignore

```diff
@@ -0,0 +1,5 @@
+content
+docs
+dist
+scripts
```

Comment:
Oh that because there is a compiled file, that is specified in .gitignore (that's why I like to use gitignore directly...)